contrib/python/api/tests/tests.py

```diff
@@ -220,6 +221,26 @@ def test_topology_rules(self):
                 "G.E().Has('RelationType', 'layer2', 'EdgeName', 'my_edge')")
         self.assertEqual(len(edge), 1, "should find only one edge")
 
-        restclient.edgerule_delete(edgerule.uuid)
+        noderules = restclient.noderule_list()
+        self.assertGreaterEqual(len(noderules), 1, "no noderules found")
```

Comment:
can you change this to 2, since we are creating two noderule it is better to check >=2

Reply:
Done